tests/benchmark/config.json

```diff
@@ -1,28 +1,64 @@
 {
   "databases": [
     {
-      "name": "postgres",
-      "params": {
-        "metadata" : {
-          "database": "postgres",
-          "schema": "postgres"
-        },
-        "conn_id": "postgres_conn"
-      }
+    "name": "bigquery",
+    "params": {
+      "metadata" : {
+        "database": "astronomer-dag-authoring",
+        "schema": "tmp_astro"
+      },
+      "conn_id": "bigquery"
     }
+  }
   ],
   "datasets": [
     {
-        "name": "few_kb",
-        "size": "45K",
-        "path":"/tmp/covid_overview.csv",
-        "rows": 732
+      "name": "ten_kb",
+      "size": "10 KB",
+      "path": "gs://astro-sdk/benchmark/trimmed/covid_overview/covid_overview_10kb.parquet",
+      "rows": 385817,
+      "conn_id": "bigquery",
+      "file_type": "parquet"
     },
     {
-        "name": "many_kb",
-        "size": "472K",
-        "path": "/tmp/artist_data.csv",
-        "rows": 3532
+      "name": "hundred_kb",
+      "size": "100 KB",
+      "path": "gs://astro-sdk/benchmark/trimmed/tate_britain/artist_data_100kb.csv",
+      "rows": 385817,
+      "conn_id": "bigquery",
+      "file_type": "csv"
+    },
+    {
+      "name": "ten_mb",
+      "size": "9.9M",
+      "path": "gs://astro-sdk/benchmark/trimmed/imdb/title_ratings_10mb.csv",
+      "rows": 385817,
+      "conn_id": "bigquery",
+      "file_type": "csv"
+    },
+    {
+      "name": "one_gb",
+      "size": "1G",
+      "path": "gs://astro-sdk/benchmark/trimmed/stackoverflow/stackoverflow_posts_1g.ndjson",
+      "rows": 385817,
+      "conn_id": "bigquery",
+      "file_type": "ndjson"
+    },
+    {
+      "name": "five_gb",
+      "size": "5G",
+      "path": "gs://astro-sdk/benchmark/trimmed/pypi/",
+      "rows": 385817,
+      "conn_id": "bigquery",
+      "file_type": "ndjson"
+    },
+    {
+      "name": "ten_gb",
+      "size": "10G",
+      "path": "gs://astro-sdk/benchmark/trimmed/github/github-archive/",
+      "rows": 385817,
+      "conn_id": "bigquery",
```

Comment:
So, does it mean `conn_id` can be a list of different syncs?

Reply:
I would just remove `conn_id` from datasets completely and just do a cross product of Datasets X Databases for benchmark. What do you think?

Fine for now, feel free to merge it -- but it will become clearer when we start benchmarking [Snowflake](https://github.com/astronomer/astro-sdk/issues/438), and [Postgres](https://github.com/astronomer/astro-sdk/issues/431)